dev_guide/quota.adoc

```diff
@@ -11,59 +11,105 @@ toc::[]
 
 == Overview
 
-A *ResourceQuota* object enumerates hard resource usage limits per project. It
-limits the total number of a particular type of object that may be created in
-a project, and the total amount of
-link:compute_resources.html[compute resources]
+A `ResourceQuota` defines constraints that limit aggregrate resource consumption
+per project.  It can limit the quantity of objects that can be created in a project
+by type, as well as the total amount of link:compute_resources.html[compute resources]
 that may be consumed by resources in that project.
```

Comment:
I'd suggest removing resources word, we're having 3 already in 2 sentences, I especially don't like this 'compute resources consumed by resources', my proposal is:

total amount of compute resources that may be consumed in that project